packages/eas-cli/src/commands/release/create.ts

```diff
@@ -0,0 +1,97 @@
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import log from '../../log';
+import { findProjectRootAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+import { getProjectIdAsync } from '../../submissions/commons';
```

Comment:
Can be done in a separate PR, but let's move this to `project/projectUtils` because it's not only for submissions and rename it to `getAppIdAsync`?